circuit/types/field/src/helpers/to_bits.rs

```diff
@@ -96,9 +96,11 @@ mod tests {
     fn check_to_bits_le(mode: Mode) {
         let expected_number_of_bits = console::Field::<<Circuit as Environment>::Network>::size_in_bits();
 
+        let mut rng = TestRng::default();
```

Comment:
ditto

Reply:
this one was actually fine, as the consecutive calls were for `check_bits_le`